extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -551,6 +626,30 @@ export class SystemLayer {
     });
   }
 
+  /**
+   * Reacts to the presence of playables on a page to determine if should disable
+   * or not the play/pause button.
+   * @param {boolean} pageHasPlayable
+   * @private
+   */
+  onPageHasPlayableStateUpdate_(pageHasPlayable) {
+    this.vsync_.mutate(() => {
+      pageHasPlayable
+        ? this.getShadowRoot().setAttribute(
+            CURRENT_PAGE_HAS_PLAYABLE_ATTRIBUTE,
+            ''
+          )
+        : this.getShadowRoot().removeAttribute(
+            CURRENT_PAGE_HAS_PLAYABLE_ATTRIBUTE
+          );
```

Comment:
Can we remove these and rely on the presence of the `disabled` attribute instead?